src/pathfinding_service/api.py

```diff
@@ -194,10 +194,14 @@ def post(self, token_network_address: str) -> Tuple[dict, int]:
             )
 
         # Create a feedback token and store it to the DB
-        feedback_token = create_and_store_feedback_tokens(
-            pathfinding_service=self.pathfinding_service,
-            token_network_address=token_network.address,
-            routes=paths,
+        feedback_token = (
+            create_and_store_feedback_tokens(
+                pathfinding_service=self.pathfinding_service,
+                token_network_address=token_network.address,
+                routes=paths,
+            )
+            if paths
+            else None
```

Comment:
How is `feedback_token.uuid.hex` on `None` gonna work?

Also,  this probably needs adjustments for the clients. Should we add this to the V2 API?

Reply:
You're totally right about the `feedback_token.uuid.hex`, of course.

About the client compatibility: I didn't expect the client to look at the feedback token when not route is present. But that might not be the case.